dsgrid/config/dimensions.py

```diff
@@ -134,6 +150,22 @@ def check_name(cls, name):
             )
         return name
 
+    @validator("display_name")
+    def check_display_name(cls, display_name):
+        if display_name == "":
+            raise ValueError(f'Empty name field for dimension: "{cls}"')
+        if REGEX_VALID_REGISTRY_DISPLAY_NAME.search(display_name) is None:
+            raise ValueError(f"display_name={display_name} does not meet the requirements")
+        return display_name
+
+    @validator("query_name")
+    def check_query_name(cls, query_name, values):
+        if "display_name" not in values:
```

Comment:
I don't understand this check, I thought `display_name` is a required field and checked prior to query_name.